pom.xml

```diff
@@ -94,6 +94,11 @@
             <id>flyway-repo</id>
             <url>s3://flyway-repo/release</url>
         </repository>
+        <repository>
+            <id>git-tzolov</id>
+            <name>tzolov's Git based repo</name>
+            <url>https://github.com/tzolov/maven-repo/raw/master/</url>
+        </repository>
```

Comment:
Now look you can download from https://github.com/greenplum-db/gpclients/tree/master/jdbc, but I can't find mvn repo for this